draft-sheffer-oauth-rfc8725bis.md

```diff
@@ -467,16 +472,15 @@ without new algorithm identifiers being required.
 
 ## Validate All Cryptographic Operations {#validate-crypto}
 
-
- All cryptographic operations used in the JWT  MUST be
- validated and the entire JWT  MUST be rejected
+All cryptographic operations used in the JWT MUST be validated and the entire JWT MUST be rejected
 if any of them fail to validate.
 This is true not only of JWTs with a single set of Header Parameters
 but also for Nested JWTs in which both outer and inner operations  MUST be validated
 using the keys and algorithms supplied by the application.
 
+Libraries MUST allow the verifier to distingush between JWS-signed and JWE-encrypted JWTs, and to only accept the former.
 
-
+When some claims of the JWT require confidentiality protection, it is RECOMMENDED to nest encrypted JWEs within signed JWSs instead of vice versa.
```

Comment:
```suggestion
```

This is unrelated to the other changes in this PR.  Please remove it.

Reply:
For the record, the following in RFC 7519 convinced me:

> Note that potential concerns about security issues related to the order of signing and encryption operations are already addressed by the underlying JWS and JWE specifications; in particular, because JWE **only supports the use of authenticated encryption algorithms**, cryptographic concerns about the potential need to sign after encryption that apply in many contexts do not apply to this specification.